.github/workflows/ci-e2e.yml

```diff
@@ -1,229 +1,49 @@
+#
+# This workflow runs CI E2E tests with Cypress.
+#
+# It relies on the container image built by 'container-images-ci.yml'.
+#
 name: E2E CI
 
 on:
     pull_request:
-    push:
```

Comment:
We do not need to run this on `master` anymore

Reply:
I think it's still sensible to run the tests on master - two branches may pass tests individually, but when both merged they may cause a failure

Reply:
> I think it's still sensible to run the tests on master - two branches may pass tests individually, but when both merged they may cause a failure

I agree and I'm 👍 to change this. I just want to point out that this PR didn't change the previous behaviour as the Cypress E2E tests matrix was only running on non-`master`

Reply:
Ahhh ok sure, makes sense then 👍

Reply:
A CI failure will anyway not block the deployment, is it ok for you?